goverance.md

```diff
@@ -0,0 +1,67 @@
+
+## Working Group Roles
+
+The SAFE Working group aims to have three Chairs, Technical Leads for each substantive block of work, and many members.
+
+Unless otherwise stated, individuals are expected to be responsive and active within
+their roles.  Within this section "member" refers to a member of a Chair, Technical Lead or
+Members.
+
+### Role of Members
+- The primary role of a member is to contribute expertise to the group.
+- Initial members are defined at the founding of the WG.
+- To add yourself as a member, submit a PR adding yourself to the list of members.
+- Members *SHOULD* remain active and responsive in their Roles.
+- Members taking an extended leave of 1 or more months *SHOULD* coordinate with other members to ensure the role is adequately staffed during the leave.
+- Members going on leave for 1-3 months *MAY* work with other members to identify a temporary replacement.
+- Members of a role *SHOULD* remove any other members that have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month. This may be done through a [super-majority] vote of members, or if there are not enough *active* members to get a super-majority of votes cast, then removal may occur through a [super-majority] vote between Chairs and Technical Leads.
+- Membership disagreements may be escalated to the WG Chairs.  WG Chair membership disagreements may be escalated to the CNCF Steering Committee.
```

Comment:
Should the chair role be defined before this section, since it's referenced here?

Reply:
I like the idea of keeping it first since it will be the most common role and putting it first feels democratic. Would it be better if the first reference to the Chairs and Tech Leads indicates that those roles are described later?

Reply:
Perhaps it makes sense to revise the "Working Group Roles" section to list the types of roles (ie in a bulleted list) and links to the relevant description section - then we will have clearly stated the group's roles before this section, and I think it will seem less odd.

Suggested edit:

----

## Working Group Roles
The SAFE Working Group is a community created with the goal of exploring secure access, policy control and safety for operators, administrators, developers, and end-users across the cloud native ecosystem.

The governance of the SAFE Working Group is comprised of:
- Three [Chairs](governance.md#role-of-chairs)
- [Technical Leads](governance.md#role-of-technical-leads) for each substantive block of work
- Group [Members](governance.md#role-of-members)

The group may have many members. Within this document, "member" may refer to a Chair, a Technical Lead, or a Member.

Reply:
I like the links. I'll make that change.